apps/src/applab/designElements/library.js

```diff
@@ -159,6 +165,40 @@ export default {
     throw new Error('unknown element type');
   },
 
+  /**
+   * Gets the theme values for this element type (if specified).
```

Comment:
This comment implies that we would sometimes want to call getThemeValues without any input, but getElementType (first line of getThemeValues) called with no input would throw an error.

Reply:
I think it will just return `undefined`, I don't think it will throw.

Reply:
Running this in the chrome console:
element.tagName.toLowerCase()
VM53:1 Uncaught ReferenceError: element is not defined
    at <anonymous>:1:1